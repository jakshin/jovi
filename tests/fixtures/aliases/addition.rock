say 1 + 1
say 2 plus 2
say "You" with "Me"
