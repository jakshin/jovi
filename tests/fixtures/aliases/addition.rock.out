2
4
YouMe
